apps/deploy-web/src/components/alerts/AlertsLayout.tsx

```
@@ -0,0 +1,66 @@
+  returnable?: boolean;
+};
+
+export const AlertsLayout: React.FunctionComponent<Props> = ({ children, page, title, headerActions, returnable }) => {
+  const router = useRouter();
+
+  const handleTabChange = (newValue: string) => {
+    switch (newValue as AlertTabs) {
+      case AlertTabs.ALERTS:
+        router.push(UrlService.alerts());
+        break;
+      case AlertTabs.CONTACT_POINTS:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)